modules/core/shared/src/main/scala/scaladex/core/api/artifact/ArtifactEndpoints.scala

```diff
@@ -0,0 +1,31 @@
+package scaladex.core.api.artifact
+
+import endpoints4s.algebra.Endpoints
+import endpoints4s.algebra.JsonEntitiesFromSchemas
+
+trait ArtifactEndpoints extends Endpoints with JsonEntitiesFromSchemas {
+
+  implicit val artifactEndpointResponse: JsonSchema[ArtifactResponse] =
+    field[String]("groupId")
+      .zip(field[String]("artifactId"))
+      .xmap[ArtifactResponse] { case (groupId, artifactId) => ArtifactResponse(groupId, artifactId) } {
+        artifactResponse => (artifactResponse.groupId, artifactResponse.artifactId)
+      }
+
+  val artifactEndpointParams: QueryString[ArtifactParams] = (qs[Option[String]](
+    name = "language",
+    docs = Some(
+      "Filter the results matching the given language version only (e.g., '3', '2.13', '2.12', '2.11', 'java')"
+    )
+  ) & qs[Option[String]](
+    name = "platform",
+    docs = Some("Filter the results matching the given platforms only (e.g., 'jvm', 'sjs1', 'native0.4', 'sbt1.0')")
+  )).xmap((ArtifactParams.apply _).tupled)(Function.unlift(ArtifactParams.unapply))
+
+  // Artifact endpoint definition
+  val artifact: Endpoint[ArtifactParams, Seq[ArtifactResponse]] =
+    endpoint(
+      get(path / "api" / "artifacts" /? artifactEndpointParams),
+      ok(jsonResponse[Seq[ArtifactResponse]])
```

Comment:
Currently the json  response looks like this.

```json
[
  {
    "groupid": "ch.epfl.scala",
    "artifactid": "scala-debug-adapter_2.13"
  },
 {
    "groupid": "ch.epfl.scala",
    "artifactid": "scalafix_2.13"
 }
]
```
That's perfect but we will probably need to add paging later on this endpoint and it will break the compatibility.
```json
{
  "currentPage": 1
  "pageCount": 10
  "totalSize": 20
  "items": [
    {
      "groupid": "ch.epfl.scala",
      "artifactid": "scala-debug-adapter_2.13"
    },
   {
      "groupid": "ch.epfl.scala",
      "artifactid": "scalafix_2.13"
   }
  ]
}
```

To make things easier for later I suggest to return `Page[ArtifactResponse]]` rather than `Seq[ArtifactResponse]` even if we don't implement paging right now (the first page contains all the `ArtifactResponse`).